### ISSUE TITLE
Request to unblock 333eth.io

### ISSUE THREAD

**Issue by Revolution333** [MetaMask/eth-phishing-detect#2429]
Good day, MetaMask blocking access to 333eth.io becouse "believes this domain could currently compromise your security and has prevented you from interacting with it" ... We are sure that 333eth.io have been compromised by a malicious actor!
 
Few days ago, admin 409H on GitHub is one, who actioned the blacklist of 333eth.io Reasoning: "Guaraneteed daily returns usually indicates a scam/ponzi scheme. I actively block these".

Sorry, but "usually" isnt correct term & 
not legally significant! 

Such actions to block the site are groundless, jeopardized the work of the project, has led to serious consequences & prevented normal work of the projct.

Many antivirus program (like Kaspersky Antivirus, Avast, ect.), crypt wallets and browser extensions (like MetaMask), take info from GitHub blacklist & block access to 333eth.io, like "fishing site"!

More than 1500 people around the world, who entered data into smart-contract more than 2500 Ether*, now can not access the site 333eth.io !!!

Dosn't exist a court order or other legal basis (document) to block  the333eth.io. The administration GitHub will to bear responsibility for all the consequences after adding 333eth.io to blacklisting.

All about situation:
https://github.com/MetaMask/eth-phishing-detect/issues/2374

Official position admin "333ETH":

1. Project "333 ETH" is not a "Ponzi Sheme"* because there is no person, no a company (businessman), who pays profits to earlier investors using funds obtained from newer investors"...

It's smart-contract algorithm (program code) on which the project is based is uploaded to the blockchain Ethereum 03.09.18 with the OWNERSHIP WAIVER function ("REFUSAL of OWNERSHIP"). This means that nobody - including the organizers- can access it to tamper with its independent functioning.

2. Admins does not have access to the main purse of the contract, it does not have the opportunity to withdraw funds from the smart-contract. Admins does not engage in the rescheduling of the ether* among the participants - it's function of the independent smart-contract.

3. Project "333 ETH" isn't working with world currency of exchange (like USD or Euro), smart-contract "333 ETH" is distribution of Ether* between participants (ether can be transferred between accounts and used to compensate participant mining nodes for computations performed). 

4. Developers of 333eth.io yesterday 14.09. opened his source-code of smart contract for analising by each one!

https://etherscan.io/address/0x311f71389e3de68f7b2097ad02c6ad7b2dde4c71

https://github.com/Revolution333/333eth

Resume:
How you can see, Project "333 ETH" not a scam or ponzi scheme &doesnt carry a threat of phishing! 333eth.io have been compromised by a malicious actor!

With Kaspersky Antivirus was the same problem. At our request, Kaspersky has already eliminated the blocking site 333ETH.io

We ask you to take all necessary measures to unblock the 333eth.io and exclude from the blacklist. Thanks!

**Comment by Mavrmich** [comment]
We hope for  unblock 333eth.io!!!

**Comment by xlab** [comment]
Copying from https://github.com/MetaMask/eth-phishing-detect/issues/2424

@409H The only unethical thing there is to block other people's work just because of an opinion of a single man, based on his life views. Without appeals or even strict guidelines what is forbidden and what is not.

The contract is a crypto game, the same as a crypto casino. You know that casinos exist and that a typical casino takes money from half of people and re-distributes it among the other half and 1% goes to the casino owners? Isn't that a scam thing and unethical, based on your views?

The contract works, there is the source published and it works straightforward, it's far more open than a typical crypto casino. People are interacting with it on their own will. I personally lost $400 on CryptoKitties, just because the kitties I invested in lost their value due to new feature announces from the game creators, so what now? I'm dealing with it, but believe me that was far more UNFAIR.

Please stop broadcasting your life views to the public by taking action on the other people's projects that are trying to make games using Ethereum. Alternatively, please provide guidelines for the project owners, so the could put **appropriate notices** on their website, to avoid confusion among the users.

@409H I hope you understand the situation better 👍 
The repo is called "eth-phishing-detect" by the way, I don't think the issues there should address which businesses are Good or Bad, this is far beyond the responsibility.

**Comment by dAppBridge** [comment]
Agree now the contract is verified it should be opened up (Unless of course reading the contract finds a problem - haven't reviewed it myself).

Perhaps the phishing protection within metamask needs improving so it can flag up contracts that are high-risk, contracts that are closed-source and unverified as well as pure phishing issues.  The one warning fits all doesn't work well in these cases.

**Comment by Mavrmich** [comment]
Dear, 409H..... please answer to our request!  We already gives you all info about situation with 333eth.io ! Please make your goodwill and unblock from blacklist! Today you have NO REASON for block it! Thk you

**Comment by dimensionless13** [comment]
Here is official Metamask twitter account post:
https://twitter.com/metamask_io/status/956273538205335554

So why just one of Metamask phishing team member decides which project is "scam" and which is not?
Why this member breaks official Metamask position? It's completely hypocrite approach.

We request immediate blocking removal!

**Comment by Mavrmich** [comment]
One day more! Admins and moderator of this topic! What are you want? Where are you? Peoples need truth! Peoples need 333eth.io by metamask! Are you hearing me?

**Comment by 777byNox777** [comment]
I conclude that the metamask is extremely engaged service influenced by and capable of blocking any site that does not like it without any reason, three thousand people already know about it and are slowly learning to use other services, the tsyphus will grow with every day 23 days ago there were 0 people now already more than three thousand, there is such a saying "do not swarm another grave into it yourself", so draw conclusions from the Lord.

**Comment by 409H** [comment]
Closing. Duplicate of https://github.com/MetaMask/eth-phishing-detect/issues/2374
